Formula/wget.rb

```diff
@@ -6,6 +6,8 @@ class Wget < Formula
   sha256 "4f4a673b6d466efa50fbfba796bd84a46ae24e370fa562ede5b21ab53c11a920"
   revision 1
 
+  env :std
```

Comment:
What's going on with `wget` here?

Reply:
@DomT4 Nothing to see here, totally didn't push an unrelated change 😆

Reply:
😸 I spent a minute reading through the `wget` formula after this PR popped up like I'm sure there's no requirement there 🙈.

Reply:
@DomT4 Was my testing https://github.com/Homebrew/brew/pull/3650

Reply:
Ah 👍. Seems like a reasonable change.